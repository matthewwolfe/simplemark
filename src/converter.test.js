const { convertToHtml } = require('./converter');

describe('converter', () => {

  test('converts markdown file to html', () => {
    const html = convertToHtml({
      markdown: '# Header 1 { .class-1 .class-2 }\n## Header 2 { .class-3 .class-4 }',
      globalCssPath: ''
    });

    expect(html).toMatchSnapshot();
  });
});
