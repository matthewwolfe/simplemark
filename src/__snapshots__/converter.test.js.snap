// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`converter converts markdown file to html 1`] = `
"<html>
  <head>
  </head>
  <body>
    <h1 class=\\".class-1 .class-2\\">Header 1 </h1>
    <h2 class=\\".class-3 .class-4\\">Header 2 </h2>
  </body>
</html>"
`;
